src/site/content/en/lighthouse-accessibility/color-contrast/index.md

```diff
@@ -1,47 +1,100 @@
+Lighthouse can't check the color contrast of
+text superimposed on an image.
+
+{% Aside 'caution' %}
+In version 2.1, WCAG expanded its color contrast requirements to
+[include user interface elements and images](https://www.w3.org/TR/WCAG21/#non-text-contrast).
+Lighthouse doesn't check these elements, but you should do so manually
+to ensure your entire site is accessible to people with low vision.
+{% endAside %}
+
+{% include 'content/lighthouse-accessibility/scoring.njk' %}
+
+## How to ensure text has sufficient color contrast
 
-## How to fix this problem
+Make sure all text on your page meets the minimum color contrast ratios
+<a href="https://www.w3.org/TR/WCAG21/#contrast-minimum" rel="noopener">specified by WCAG</a>:
+- 3:1 for text that is 18&nbsp;pt, or 14&nbsp;pt and bold
+- 4.5:1 for all other text
 
-To fix this problem,
-provide sufficient contrast for background and foreground colors.
-Some people with low vision experience low contrast.
-Everything tends to appear about the same brightness,
-which makes it hard to distinguish outlines, borders, edges, and details.
+One way to find a color that will meet contrast requirements
+is to use Chrome DevTools' color picker:
+1. Right-click (or `Command`-click on Mac) the element you want to check,
+   and select **Inspect**.
+1. In the **Styles** tab of the **Elements** pane,
+   find the `color` value for the element.
+1. Click the color thumbnail next to the value.
 
-Text that is too close in luminance (brightness) to the background can be hard to read,
-especially for people with low vision,
-but all users can benefit from sufficient contrast.
-Ensure color contrast of at least 4.5:1 for small text or 3:1 for large text.
-Large text is defined as 18pt or 14pt bold.
+The color picker tells you whether the element
+meets color contrast requirements,
+taking font size and weight into account:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="color-picker.png"
+    alt="Screenshot of Chrome DevTools color picker with color contrast information highlighted">
+</figure>
+
+You can use the color picker to adjust the color
+until its contrast is high enough.
+It's easiest to make adjustments in the HSL color format.
+Switch to that format by clicking the toggle button on the right of the picker:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="color-format-toggle.png"
+    alt="Screenshot of Chrome DevTools color picker with the color format toggle highlighted">
+</figure>
 
-Try the Color Contrast Analyser in
-[Text elements must have sufficient color contrast against the background](https://dequeuniversity.com/rules/axe/3.3/color-contrast).
+Once you have a passing color value, update your project's CSS.
 
-<!--
-## How this audit impacts overall Lighthouse score
+More complex cases, like text on a gradient or text on an image,
+need to be checked manually,
+as do UI elements and images.
+Consider using a tool like the Paciello Group's
```

Comment:
True for HTML text on an image, but I was wanting to provide a solution for raster text and other images and was worried about length/complexity. To use the color picker eyedropper in the latter cases requires you to first inspect some HTML text and then eye drop the image you're actually interested in, which is pretty cumbersome.

I could break out the HTML-text-on-an-image case, but I kinda think presenting one tool that you can use for multiple cases is more straightforward. But happy to do whatever you think best here!

Reply:
I didn't know their tool handled raster text, or essentially image over image. That's neat.
Personally I think it's useful to give a shout out to the eyedropper because positioning text over an image with CSS is pretty common, and having to install a native app to check that contrast feels a bit burdensome.

Kind of a bummer that we don't mention that functionality here: https://developers.google.com/web/tools/chrome-devtools/accessibility/reference#contrast
I know Alice spent a decent amount of time building it, which is why I wanted to highlight it.

Is it possible to add an Aside note to mention that you can use the eyedropper in devtools, or the Pacielllo app, to test against image backgrounds?

Reply:
> I didn't know their tool handled raster text, or essentially image over image. That's neat.

I mean, it just grabs the color from any pixel you point it at.

> Personally I think it's useful to give a shout out to the eyedropper because positioning text over an image with CSS is pretty common, and having to install a native app to check that contrast feels a bit burdensome.

I take your point, though I personally find it less burdensome to install a small native app that can check everything than to use multiple tools for slightly different cases. That said, I'll put in a note about the background color picker!